src/plugin/plugin.rs

```diff
@@ -74,7 +74,7 @@ where
     }
 
     /// Provided for use when staging systems outside of this plugin using
-    /// [`with_system_setup(false)`](Self::with_system_setup).
+    /// [`with_system_setup(false)`](Self::with_default_system_setup).
```

Comment:
Is this correct or should it also be `with_default_system_setup(false)`?

Reply:
I'll look at it ; there are still a few warnings left, but I see the ci step is green, I'll see if I can make it fail on warnings